src/creature.cpp

```diff
@@ -1027,6 +1027,11 @@ bool Creature::resists_effect( const effect &e )
     return false;
 }
 
+const pimpl<effects_map> &Creature::get_effects() const
```

Comment:
If you need to expose this, expose the `effect_map` directly. Nobody needs to know about the `pimpl`.